pkg/tfbridge/schema.go

```diff
@@ -1022,33 +1022,61 @@ func CleanTerraformSchema(tfs map[string]*schema.Schema) map[string]*schema.Sche
 }
 
 // CoerceTerraformString coerces a string value to a Go value whose type is the type requested by the Terraform schema
-// type. Returns an error if the string can't be successfully coerced to the requested type.
-func CoerceTerraformString(schType schema.ValueType, stringValue string) (interface{}, error) {
+// type or the Pulumi SchemaInfo. We prefer the SchemaInfo overrides as it's an explicit call to action over the
+// Terraform Schema. Returns an error if the string can't be successfully coerced to the requested type.
+func CoerceTerraformString(schType schema.ValueType, schemaOverride *SchemaInfo, stringValue string) (interface{}, error) {
```

Comment:
Very minor - but we call this `*SchemaInfo` `ps` almost everywhere else in this file - might make sense to keep the naming the same here.  (whether that's the best name anywhere is certainly debatable - but consistency...)

Reply:
done :)